test/ably/resthttp_test.py

```diff
@@ -135,3 +137,14 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_version_headers(self):
+        # RSC7a, RSC7b
```

Comment:
RSC7b says it supports variants? http://docs.ably.io/client-lib-development-guide/features/#RSC7b